auth/src/main/java/com/firebase/ui/auth/ui/email/EmailActivity.java

```diff
@@ -130,4 +134,14 @@ private void setSlideAnimation() {
         // Make the next activity slide in
         overridePendingTransition(R.anim.fui_slide_in_right, R.anim.fui_slide_out_left);
     }
+
+    private void setUpTermsOfServiceFooter() {
```

Comment:
I'm seeing this method a lot. I wonder if it makes sense to add a `PrivacyDisclosureUtils` class that includes this and the tos methods you added in `FlowParameters` (I'm not a huge fan of logic in a POJO). @samtstern What do you think?

Reply:
Seems like a good idea to me, less repetition is better!